luke_bot/smashgg_query.py

```diff
@@ -239,12 +251,10 @@ def ongoing_results(event_id: int, entrant_id: int):
             nodes{
                 fullRoundText
                 displayScore
-                slots{
-                    entrant{
-                        id
-                        name
-                    }
-                }
+                wPlacement
```

Comment:
amended query to delete excess info we won't be using anytime soon in favour of placement metadata that we can eventually add